src/main/java/org/broadinstitute/consent/http/service/DatabaseElectionAPI.java

```diff
@@ -701,5 +706,18 @@ private void updateFinalVote(Election rec, Integer electionId) {
         }
     }
 
-
+    private void sendResearcherNotification(String referenceId) throws Exception {
+        BasicDBObject query = new BasicDBObject(DarConstants.ID, new ObjectId(referenceId));
+        Document dar = mongo.getDataAccessRequestCollection().find(query).first();
+        List<Integer> dataSetIdList = dar.get(DarConstants.DATASET_ID, List.class);
```

Comment:
I've seen this pattern a few times using an unsafe cast of `List` to `List<T>`. We should try to wrap this assignment into a cast-safe utility method somewhere and add some TODOs about using it.